### PR TITLE
Use mod.conf for name/dependencies

### PR DESCRIPTION
**Pull request by Panquesito7**
Deletes deprecated `depends.txt`.

### CODE REVIEW

**Comment by Panquesito7** [comment]
Ping?

**Comment by Panquesito7** [comment]
@sofar.

**Comment by appgurueu** [comment]
This will break compatibility with 0.4.x

File: mod.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name = frameglass
```
Comment (appgurueu) [inline mod.conf:1]: framedglass*

**Review by appgurueu — CHANGES_REQUESTED** [review]

File: mod.conf (hunk 1 above)
Comment (BuckarooBanzay) [reply mod.conf:1]: oh, wow, thanks for spotting that, completely missed that :+1:  @Panquesito7 do you have time for this? I could open another PR with the changes if you want...

**Comment by BuckarooBanzay** [comment]
> This will break compatibility with 0.4.x

Is that such a bad thing?

**Comment by appgurueu** [comment]
> > This will break compatibility with 0.4.x
> 
> Is that such a bad thing?

For such an old mod which otherwise preserves compatibility? Maybe. There's little to gain. Why must `depends.txt` be removed?

**Comment by BuckarooBanzay** [comment]
IMO: older server-operators can still use the old version (most of them don't even upgrade the mods anyway)

> There's little to gain. Why must depends.txt be removed?

Basically: consistency with every other mod, there is no additional functionality gained (for now)

**Pull request merged**